.cci.jenkinsfile

```diff
@@ -89,6 +90,10 @@ parallel vmcheck: {
           """)
           archiveArtifacts allowEmptyArchive: true, artifacts: 'vmcheck-logs.tar.xz'
         }
+      },
+      kola: {
+        fcosKola("${env.WORKSPACE}")
```

Comment:
```suggestion
        fcosKola(cosaDir: "${env.WORKSPACE}")
```

Reply:
That said, this will easily blow through the requested resources. I think we'd need to bump `nhosts` to 6+8+1 (+8 for the regular "internal" tests being run with `--parallel 8` and +1 for the upgrade test). Let's maybe just do it serially instead? (Or alternatively, let's *only* run our external tests here since we already have our own much more in-depth host tests).

Reply:
>  (Or alternatively, let's only run our external tests here since we already have our own much more in-depth host tests).

An huge and interesting topic there.  I think what I'd like to do is add support for "tags" to kola tests, so we can do like `kola run --tags ostree` and only run tests that are related to ostree by default or so.

Reply:
I think that makes sense. We might already be able to do that if something like `*ostree*` works today.

Reply:
Tags would be an awesome UX improvement over the current regex globbing. Definitely throw up a ticket, I might have some cycles I'd like spending on it in the near future.

Reply:
https://github.com/coreos/coreos-ci-lib/pull/20